virttest/standalone_test.py

```diff
@@ -21,14 +21,18 @@ def _variant_only_file(filename):
                       if not _.lstrip().startswith('#')])
 
 
+#: The first entry of these lists will be used as a default value.
+#: Only change the first entry if you intend to change the default
+#: value.  See :func:`avocado_vt.plugins.vt.add_basic_vt_options` and
+#: class:`avocado_vt.options.VirtTestOptionsProcess` for usage examples.
```

Comment:
How about the order of the items after the first one? I guess it is free of restrictions then?

Reply:
Yes, there's nothing depending on the order following the first entry AFAICT (and it's not documented).